client/query/queryActions.js

```diff
@@ -46,10 +49,14 @@ export const receiveEvents = (events) => ({
   },
 });
 
-export const updateEvents = (events, dest) => {
-  const addedEvents = events.filter(event => event.addedToDest);
+export const reset = () => ({
+  type: 'RESET',
+});
+
+export const updateEvents = (events, dest, goNext, destIdx) => {
+  const addedEvents = events[0].filter(event => event.addedToDest);
```

Comment:
This is `events[0]` because the events returned from the placeSearch.search function is an array of arrays of events (to account for multiple destinations). Since our updateEvents is called on only 1 destination at a time, the array of arrays sent back has a length of 1, and we're only interested in that first array of events.